docs/faq.md

```diff
@@ -1,5 +1,137 @@
 # Frequently Asked Questions
 
+## Driver performance for large-scale clusters
+
+### Summary of v1.25 scalability improvements
```

Comment:
This is such a large "question" should it have a dedicated `scalability.md` instead of being in the FAQ?

Reply:
Today, because we're adding our FAQ to our README links, I think we should keep this info in our FAQ.

I think this question should be re-raised once our FAQ has enough content (and enough Cxs have discovered it) to warrant splitting parts of it off.